packages/manager/modules/dedicated-universe-components/src/directives/sshkeySwitch/sshkeySwitch.js

```diff
@@ -1,4 +1,4 @@
-angular.module('UserAccount').directive('sshkeySwitch', [
+export default /* @ngInject */ [
   'sshkey-regex',
   function sshkeySwitchDirective(SSHKEY_REGEX) {
```

Comment:
As we `@ngInject` I don't think the array is necessary anymore
```suggestion
export default
  /* @ngInject */    function sshkeySwitchDirective  (SSHKEY_REGEX) {
```